src/platforms/rust/index.mdx

```diff
@@ -63,6 +63,14 @@ and libraries.
 A list of integrations and their feature flags can be found
 [in the integration API docs](https://docs.rs/sentry/0/sentry/integrations/index.html).
 
+<Note>
+
+If you are not stripping down or splitting debug information files, but rather keep them in the produced binary instead,
+make sure to not add `debug_images` integration and it's corresponding `debug-images` feature flag,
+as it will collide with the built-in debug files processing.
```

Comment:
```suggestion
If you keep debug information files in the produced binary, rather than strip them down or split them,
do not add the `debug_images` integration and the corresponding `debug-images` feature flag,
as it will collide with the built-in debug file processing.
```